test/unit/world_location_test.rb

```diff
@@ -28,68 +31,56 @@ class WorldLocationTest < ActiveSupport::TestCase
     end
 
     should "filter out any results that don't have a slug" do
-      loc1 = world_location_details_for_slug("location-1")
-      loc2 = world_location_details_for_slug("location-2")
-      loc2["details"]["slug"] = nil
```

Comment:
I notice we're not handling this case anymore. Do we assume this can't be nil?

Reply:
The code under test doesn't make any special case for nil (uses present?) so I felt it didn't need to exercise both conditions.

Though I don't really want to pick too much at this as I think there's about 3 other conditionals that are part of this which aren't tested 🙈